Scripts/Utility/Editor/HandyFields.cs

```diff
@@ -136,8 +136,7 @@ public static void Label(string label)
         }
         public static Texture2D GetMaterialTexture(Material material)
         {
-            var tex = material.GetTexture("_BaseMap") as Texture2D;
-            if (tex != null) return tex;
```

Comment:
I thought this would work to support both built-in renderer and the scriptable render pipelines, but I guess not. I'll see if I can find a way around this.